ansible/roles/ooni-backend/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+enable_dns_helper: yes
+enable_dns_discovery_helper: yes
+enable_http_return_json_headers_helper: yes
+enable_tcp_echo_helper: yes
+enable_daphn3_helper: yes
+
+dns_resolver_address: "8.8.8.8:53"
+dns_test_helper_address: "37.218.247.110:57004"
+http_return_json_headers_address: "http://37.218.247.95"
+ssl_address:
+tcp_echo_address: "37.218.247.110"
+traceroute_address: "199.119.112.49"
+http_return_json_headers_port: 57001
+tcp_echo_port: 57002
+daphn3_port: 57003
+dns_port_tcp: 57004
+dns_port_udp: 57004
+dns_discover_port_tcp: 57005
+dns_discover_port_udp: 57005
```

Comment:
Dns discovery should not be running as we use akamai instead. Also running the DNS discovery helper on port 57005 is basically useless, since you can't configure any recursive resolve to resolve on a port other than 53.